README.md

```diff
@@ -13,15 +13,51 @@ While our first priority is a stable build, we will eventually add features that
 
 [Feature Requests](http://feathub.com/jellyfin/jellyfin)
 
-## Building Jellyfin packages
+## Prebuilt Jellyfin packages
+
+Prebuild packages are available for Debian/Ubuntu and Arch, and via Docker Hub.
+
+### Docker
+
+The Jellyfin Docker image is available on Docker Hub at https://hub.docker.com/r/jellyfin/jellyfin/
+
+### Arch
+
+The Jellyfin package is in the AUR at https://aur.archlinux.org/packages/jellyfin-git/
+
+### Debian/Ubuntu
+
+A package repository is available at https://repo.jellyfin.org. To use it:
+
+0. Install the `dotnet-runtime-2.1` package via [Microsoft's repositories](https://dotnet.microsoft.com/download/dotnet-core/2.1).
+0. Import the GPG signing key (signed by Joshua):
+    ```
+    wget -O - https://repo.jellyfin.org/debian/jellyfin-signing-key-joshua.gpg.key | sudo apt-key add -
+    ```
+0. Add an entry to `/etc/sources.list.d/jellyfin.list`:
+    ```
+    echo "deb https://repo.jellyfin.org/debian $( grep -Ewo -m1 --color=none 'jessie|stretch|buster' /etc/os-release || echo buster ) main" | sudo tee /etc/apt/sources.list.d/jellyfin.list
+    ```
+0. Update APT repositories:
+    ```
+    sudo apt update
+    ```
+0. Install Jellyfin:
+    ```
+    sudo apt install jellyfin
+    ```
+
+## Building Jellyfin packages from source
 
 Jellyfin seeks to integrate build facilities for any desired packaging format. Instructions for the various formats can be found below.
 
+NOTE: When building from source, it is strongly advised to clone the full Git repository, rather than using a `.zip`/`.tar` archive.
+
 ### Debian/Ubuntu
 
 Debian build facilities are integrated into the repo at `debian/`.
 
-1. Install the `dotnet-sdk-2.1` package via [Microsoft's repositories](https://dotnet.microsoft.com/download/linux-package-manager/debian9/sdk-2.1.500).
+1. Install the `dotnet-sdk-2.1` package via [Microsoft's repositories](https://dotnet.microsoft.com/download/dotnet-core/2.1).
 2. Run `dpkg-buildpackage -us -uc -jX`, where X is your core count.
```

Comment:
I think we can drop "-jX" part as it is not passed anywhere (at least not passed to `dotnet publish`).

Also might need a note regarding build as regular user - in my experience `dotnet publish` when run as regular user from `fakeroot` goes nuts. Or better solve that problem somehow...

Reply:
Good point on the threads.

I've been running it in fakeroot without issue, weird.

Reply:
Then it might be due to me running that in unprivileged lxc container... I'll try to drop a note about that later in readme.